source/upgrade/important-upgrade-notes.rst

```diff
@@ -16,6 +16,7 @@ Important Upgrade Notes
 .. important::
    - Support for Mattermost Server v5.37 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ is coming to the end of its life cycle in April 15, 2022. Upgrading to Mattermost Server v6.3 `Extended Support Release <https://docs.mattermost.com/upgrade/extended-support-release.html>`_ or later is required.
    - `Collapsed Reply Threads <https://docs.mattermost.com/messaging/organizing-conversations.html>`_, available in beta, are known to have a negative impact on server performance. If you cannot easily scale up and tune your database, or if you are running the Mattermost application server and database server on the same machine, we recommended disabling `ThreadAutoFollow <https://docs.mattermost.com/configure/configuration-settings.html#automatically-follow-threads>`_ and `CollapsedThreads <https://docs.mattermost.com/configure/configuration-settings.html#collapsed-reply-threads-beta>`_ until Collapsed Reply Threads is promoted to general availability in Q2 2022. Learn more about these `performance considerations here <https://support.mattermost.com/hc/en-us/articles/4413183568276>`_.
+   - MariaDB databases are not currently supported on v6.4 and v6.5. As a workaround, `this migration <https://github.com/mattermost/mattermost-server/blob/master/db/migrations/mysql/000066_upgrade_posts_v6.0.up.sql>`_ can be added to the migrations table to manually solve the problem.
```

Comment:
The mentioned workaround is a version specific one and the JSON data type thing appears on other migrations as well, so I'd suggest simply we can ask them to not upgrade at all.

Reply:
Or use MySQL ;)